src/App.tsx

```diff
@@ -1,19 +1,75 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useCallback,
+  useContext, useEffect, useMemo, useRef, useState,
+} from 'react';
+
+import { getTodos } from './api/todos';
 import { AuthContext } from './components/Auth/AuthContext';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+import { FilterBy } from './types/FilterBy';
+import { TodoNav } from './components/TodoNav';
+import { ErrorNotification } from './components/ErrorNotifications';
 
 export const App: React.FC = () => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const user = useContext(AuthContext);
   const newTodoField = useRef<HTMLInputElement>(null);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterBy, setFilterBy] = useState<FilterBy>(FilterBy.All);
+  const [hasError, setHasError] = useState(false);
+  const [todoTitle, setTodoTitle] = useState('');
+
+  const getTodosFromServer = async () => {
+    if (user) {
+      try {
+        const todosFromServer = await getTodos(user.id);
+
+        setTodos(todosFromServer);
+      } catch {
+        setHasError(true);
+
+        setTimeout(() => {
+          setHasError(false);
+        }, 3000);
+      }
+    }
+  };
+
+  const deleteErrors = useCallback(() => {
+    return setHasError(false);
+  }, []);
+
+  const visibleTodos = useMemo(() => (
+    todos.filter(todo => {
+      switch (filterBy) {
+        case FilterBy.Completed:
+          return todo.completed === true;
+        case FilterBy.Active:
+          return todo.completed === false;
```

Comment:
you don't need `=== true` and `=== false` here
todo.completed is already `true` or `false`